driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpClientChannelSink.java

```diff
@@ -261,6 +263,21 @@ protected void flushRequested(ChannelPipeline pipeline, FlushEvent evt) throws E
     }
 
     @Override
+    protected void abortRequested(ChannelPipeline pipeline, final AbortEvent evt) throws Exception {
+        HttpClientChannel channel = (HttpClientChannel) pipeline.getChannel();
+        ChannelFuture flushFuture = Channels.future(channel);
+        flushRequested(channel, flushFuture);
```

Comment:
I think an implicit flush may be better here, as I can't see a case where a script author would want to add a write followed by an abort, and not actually have the write make it to the wire.  

Note: We make this same assumption when the connection is upgraded or in a connection close streaming state and the author has a `close`.

If you agree I'll add a comment here explaining why